bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieJournalTest.java

```diff
@@ -794,4 +801,50 @@ private void testPartialFileInfoPostV3Journal(boolean truncateMasterKey)
             // correct behaviour
         }
     }
+
+    @Test(expected = IOException.class)
```

Comment:
Is is too ambiguous.
We ate not sure that only the last line threw this exception.

Please move the assertion exactly around the expected invocation

Reply:
This annotation works at a class level -- not sure how I can make it to the specific line?

Reply:
It is not at **class** level, but it works around this **method**